lib/kiev/shoryuken.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative "base"
+
+module Kiev
+  module Shoryuken
+    require_relative "shoryuken/request_store"
+    require_relative "shoryuken/request_logger"
+    require_relative "shoryuken/request_id"
+    require_relative "shoryuken/store_request_details"
+    require_relative "shoryuken/message_tracer"
+    require_relative "shoryuken/tree_path_suffix"
+
+    module MessageAttributeAccess
+      def string_value(message, key)
```

Comment:
I see the very same module here:
https://github.com/blacklane/kiev/pull/2/files#diff-72ef2b4120df7b3c2a3555e5ea4fe9a0R6
I guess this one can be safely removed.

Reply:
I'd like to move it, move it. 😐